integration-cli/docker_cli_run_test.go

```diff
@@ -146,34 +146,6 @@ func (s *DockerSuite) TestRunWorkingDirectory(c *check.C) {
 	}
 }
 
-// pinging Google's DNS resolver should fail when we disable the networking
-func (s *DockerSuite) TestRunWithoutNetworking(c *check.C) {
```

Comment:
I'm a little concerned from the Windows side if --net=none is deprecated? What's the replacement? There will be other plumbing required in the Windows drivers if it is deprecated (along with this test)